frame/election-provider-support/src/lib.rs

```diff
@@ -501,6 +500,37 @@ pub trait SortedListProvider<AccountId> {
 	/// Return true if the list already contains `id`.
 	fn contains(id: &AccountId) -> bool;
 
+	/// Get the score of `id`.
+	fn get_score(id: &AccountId) -> Result<Self::Score, Self::Error>;
+
+	/// Check internal state of list. Only meant for debugging.
+	fn try_state() -> Result<(), &'static str>;
```

Comment:
This should be feature gated by `try-runtime`.

Reply:
The thing is that this was ported from the original SortedListProvider and it definitely was not feature-gated there. And I've seen more occurrences of that in the codebase, perhaps in BagsList.

Reply:
Yep, it was missing all around bags-list. So now that's feature-gated it breaks a bunch of things. Going to be fixed as part of this PR.